src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  useRouteMatch,
+  useParams,
+} from 'react-router-dom'
+import { Test } from './components/test.component'
+import { Home } from './components/home.component'
+import { About } from './components/about.component'
+import WhoAreYou from './pages/WhoAreYou'
+import { Navbar } from './components/navbar.component'
+
+function App() {
+  return (
```

Comment:
You don't need {} return () here ;)
Can be replaced with `const App = () => ( ...components)`

Reply:
Oki :) Done!